flycheck.el

```diff
@@ -5969,7 +6075,12 @@ otherwise."
          :checker checker
          :filename (if (or (null filename) (string-empty-p filename))
                        (buffer-file-name)
-                     filename))))))
+                     filename)
+         :end-line (flycheck-string-to-number-safe end-line)
+         :end-column (flycheck-string-to-number-safe end-column)
+         :region (when (or region-start region-end)
+                   (cons (flycheck-string-to-number-safe region-start)
+                         (flycheck-string-to-number-safe region-end))))))))
```

Comment:
Should we add region-start and region-end patterns if we don't have use cases for them yet? I wouldn't expect many CLI tools that only provide a text-based interface to provide regions.